### ISSUE TITLE
[Enhancement] Possibility to schedule also by day

### ISSUE THREAD

**Issue by kmoppel** [cybertec-postgresql/pg_squeeze#31]
Currently one can only schedule by time...but the "day of week" component would also be very handy as in many cases locking can only be "afforded" during the weekend.

**Comment by janson2000** [comment]
This is a good setting. also, the task_interval column in the new version is cancelled?

**Comment by ahouska** [comment]
@janson2000 Yes, the "task_interval column" has been replaced by "schedule", see the "Release 1.1.1" section of https://github.com/cybertec-postgresql/pg_squeeze/blob/master/NEWS

@kmoppel I'll change the data type of the "schedule" column to timestamptz in the next version of pg_squeeze.

**Comment by janson2000** [comment]
hi,
"schedule", now is to set the schedule at a certain point in time. What about the following application scenario? A table, after the first squeeze, how to set 7 days as a scheduling cycle?


Thanks in advance


------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"ahouska"<notifications@github.com&gt;;
发送时间:&nbsp;2020年6月9日(星期二) 晚上10:29
收件人:&nbsp;"cybertec-postgresql/pg_squeeze"<pg_squeeze@noreply.github.com&gt;;
抄送:&nbsp;"Jason"<2065685868@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [cybertec-postgresql/pg_squeeze] [Enhancement] Possibility to schedule also by day (#31)





 
@janson2000 Yes, the "task_interval column" has been replaced by "schedule", see the "Release 1.1.1" section of https://github.com/cybertec-postgresql/pg_squeeze/blob/master/NEWS
 
@kmoppel I'll change the data type of the "schedule" column to timestamptz in the next version of pg_squeeze.

**Comment by ahouska** [comment]
Please check pg_squeeze 1.3 (currently beta): https://github.com/cybertec-postgresql/pg_squeeze/tree/REL1_3

**Comment by janson2000** [comment]
&nbsp;Roger that. thank you very much for your sharing.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "cybertec-postgresql/pg_squeeze"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年8月11日(星期二) 下午5:22
收件人:&nbsp;"cybertec-postgresql/pg_squeeze"<pg_squeeze@noreply.github.com&gt;;
抄送:&nbsp;"Jason"<2065685868@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [cybertec-postgresql/pg_squeeze] [Enhancement] Possibility to schedule also by day (#31)





 
Please check pg_squeeze 1.3 (currently beta): https://github.com/cybertec-postgresql/pg_squeeze/tree/REL1_3
